cobalt/docker/linux/Dockerfile

```diff
@@ -1,6 +1,30 @@
 # Use the official Ubuntu image as the base
 FROM debian:12
 
+# TODO(b/372303096): Move to dedicated unittest image.
+RUN echo deb http://deb.debian.org/debian-debug \
+    $(. /etc/os-release && echo "${VERSION_CODENAME}")-debug main \
+    >> /etc/apt/sources.list
+
+# TODO(b/372303096): Move to dedicated unittest image.
```

Comment:
nit: can we add this TODO to an actual code vs having to maintain 2 lists?

Reply:
Best I can do is to add a separate apt command for these. Do you prefer that?

Can't put comments in those escaped lines as bash will concat them and everything after the first comment is now a comment.

Reply:
ah never mind then, I am was just afraid two lists will go out of sync.